File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/commands/low_level/SetOuttakeClawStateCommand.java
package org.firstinspires.ftc.teamcode.commands.low_level;

import com.arcrobotics.ftclib.command.InstantCommand;

import org.firstinspires.ftc.teamcode.managers.OuttakeManager;

public class SetOuttakeClawStateCommand extends InstantCommand {

    public SetOuttakeClawStateCommand(OuttakeManager manager, OuttakeManager._OuttakeClawServoState state){
        super(()-> manager.update(state));
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/commands/low_level/outtake/SetOuttakeClawStateCommand.kt
package org.firstinspires.ftc.teamcode.commands.low_level.outtake;

import com.arcrobotics.ftclib.command.InstantCommand;
import org.firstinspires.ftc.teamcode.managers.OuttakeManager;

class SetOuttakeClawStateCommand(manager: OuttakeManager, state: OuttakeManager._OuttakeClawServoState) : InstantCommand({manager.update(state)})